client/state/reader/teams/actions.js

```diff
@@ -7,11 +7,26 @@
  */
 import {
 	READER_TEAMS_REQUEST,
+	READER_TEAMS_RECEIVE,
 } from 'state/action-types';
 
 export function requestTeams() {
 	return {
 		type: READER_TEAMS_REQUEST,
+		meta: {
+			requestStart: READER_TEAMS_REQUEST
+		}
+	};
+}
```

Comment:
@gwwar do you have any gut guesses here?  I had a couple ideas:
1. helper func possibility: serialize the whole action and use it as the requestKey 
2. potential convention: `${action_type}:${identifier}`